### ISSUE TITLE
Pushing "publier" but the site is giving a message like i was going back without saving 

### ISSUE THREAD

**Issue by MaudeFJ** [culturecreates/tech-support-tickets#24]
<img width="1323" alt="Capture d’écran, le 2023-06-22 à 09 39 54" src="https://github.com/culturecreates/tech-support-tickets/assets/135066077/7c8cdfc1-b42d-4eeb-ad9d-a62c3842725c">
Then i did cancel cause it felt like i was gonna loose my work. and instead of being publish it was still in draft.

**Comment by MaudeFJ** [comment]
It also say bravo it's publish after but still a draft

**Comment by troughc** [comment]
When this is fixed, we should also be able to close ticket https://github.com/culturecreates/footlight-app/issues/253

**Comment by saumier** [comment]
@troughc Please test in staging.

**Comment by troughc** [comment]
@saumier I am not sure how to test this one unfortunately. I am not able to reproduce it, i.e. clicking Publish and then getting that message

I now realize this issue isn't exactly the same as issue https://github.com/culturecreates/footlight-app/issues/253 



I reopened that one because I can test it (which I did). I have now moved it to To Do, and added the following info (it still needs to be fixed). 
When I don't click save or publish, I sometimes get a message (when I simply click Events in left sidebar).
![Image](https://github.com/culturecreates/tech-support-tickets/assets/64489312/3f474bbd-1165-407a-9223-1772be27e3d4)
